app/assets/javascripts/components/ReportFilter.jsx

```diff
@@ -7,23 +7,57 @@ class ReportFilter extends React.Component {
     super(props);
     this.sample_id = props.sample_id;
     this.background_model = props.background_model || 'N/A';
+    this.backgroundModels = props.all_backgrounds || [];
     this.all_categories = props.all_categories || [];
-    this.state = ReportFilter.genusSearchValueFor(props.report_page_params.selected_genus);
     this.genus_search_items = this.props.all_genera_in_sample;
     this.genus_search_items.splice(0, 0, 'None');
     this.applyExcludedCategories = this.applyExcludedCategories.bind(this);
     this.applyGenusFilter = this.applyGenusFilter.bind(this);
     this.enableFilters = this.enableFilters.bind(this);
     this.clearGenusSearch = this.clearGenusSearch.bind(this);
+    this.handleBackgroundModelChange = this.handleBackgroundModelChange.bind(this);
+    this.handleGenusSearch = this.handleGenusSearch.bind(this);
+    this.state = {
+      genusSearchValue: this.setGenusSearchValueFor(props.report_page_params.selected_genus) || "",
+      backgroundName: this.background_model.name || null,
+      backgroundParams: this.background_model.id || null
+    }
   }
 
   componentDidMount() {
+    this.initializeSelectTag();
+    $(ReactDOM.findDOMNode(this.refs.background)).on('change',this.handleBackgroundModelChange);
     // a polyfill for firefox, but disbaled for now
     // $(window).resize(() => {
     //   this.resizeFilterHeight();
     // });
   }
 
+  initializeSelectTag() {
+    $('select').material_select();
+  }
+
+  handleGenusSearch(e) {
+    this.setState({
+      genusSearchValue: this.setGenusSearchValueFor(e.target.value)
+    })
+  }
+
+  refreshPage(overrides) {
+    new_params = Object.assign({}, this.state.backgroundParams, overrides);
```

Comment:
this.state.backgroundParams? 
This seems like a bug. 
it should be props.report_page_params?